js/packages/teams-ai/src/ConversationHistory.spec.ts

```diff
@@ -245,7 +245,7 @@ describe('ConversationHistory', () => {
             assert.equal(lastLine, expectedLastLine);
         });
 
-        // TODO: When history exists, but no SAY or DO exist in the last line, replacing the entire line with `${assistantPrefix}${newResponse}` doesn't ensure that SAY is included in newResponse.
+        // TODO: (Discussion) When history exists, but no SAY or DO exist in the last line, replacing the entire line with `${assistantPrefix}${newResponse}` doesn't ensure that SAY is included in newResponse.
```

Comment:
@Stevenic, @aacebo, @singhk97, could you weigh in on this opinion?

Reply:
This is all likely to change for GA so I wouldn't get too worried about this.